lib/ui/costumes_page.dart

```diff
@@ -40,7 +40,10 @@ class CostumesPage extends StatelessWidget {
       body: ListView.separated(
         itemBuilder: (context, index) => CostumeItem(
           title: costumes[index].title,
-          onTap: null,
```

Comment:
Make sure you specify the Navigator's context beforehand, instead of using it as a argument.